### PR TITLE
Be explicit about impact of not writing caches

### PR DESCRIPTION
**Pull request by mtrmac**
A follow-up to https://github.com/containers/storage/pull/2031 .

### CODE REVIEW

**Review by flouthoc — APPROVED** [review]
LGTM

File: pkg/chunked/cache_linux.go [hunk 1]
```diff
@@ -289,8 +289,10 @@ func (c *layersCache) load() error {
 		}
 
 		if r.ReadOnly {
-			// if the layer is coming from a read-only store, do not attempt
+			// If the layer is coming from a read-only store, do not attempt
 			// to write to it.
+			// Therefore,we won’t find any matches in read-only-store layers,
```
Comment (kwilczynski) [inline pkg/chunked/cache_linux.go:294]: Missing space after the comma.

Comment (mtrmac) [reply pkg/chunked/cache_linux.go:294]: Addressed in #2088. Thanks!

**Pull request merged**